### PR TITLE
Avatar url Updated to .png Url 

### PR DESCRIPTION
**Pull request by rohan-kulkarni-25**

### CODE REVIEW

**Comment by FayasNoushad** [comment]
Good work 👍

File: public/data/Kaushik.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Kaushik",
   "bio": "Student",
-  "avatar": "https://avatars.githubusercontent.com/KaushikKC",
+  "avatar": "https://github.com/KaushikKC.png",
```
Comment (nhcarrigan) [inline public/data/Kaushik.json:4]:
```suggestion
  "avatar": "https://github.com/KaushikK.png",
```

Discrepancy between GitHub username and avatar username?

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

File: public/data/Kaushik.json (hunk 1 above)
Comment (rohan-kulkarni-25) [reply public/data/Kaushik.json:4]: `https://github.com/KaushikKC.png` This one url is correct as this person opened issue Both url are available but the one with kkc is right person

Comment (nhcarrigan) [reply public/data/Kaushik.json:4]: I think it's confusing to have the GitHub URL different from the avatar URL - maybe we should leave this one alone and allow them to update it themselves.

Comment (rohan-kulkarni-25) [reply public/data/Kaushik.json:4]: > I think it's confusing to have the GitHub URL different from the avatar URL - maybe we should leave this one alone and allow them to update it themselves.

Yeah we can do that. I will commit their own profile url which they uploaded and ask them to update it

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**